### PR TITLE
Rubocop updates

### PR DESCRIPTION
**Pull request by pyromaniac**

### CODE REVIEW

File: chewy.gemspec [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
```
Comment (barthez) [inline chewy.gemspec]: Do we need this `coding: utf-8` line?

Comment (pyromaniac) [inline chewy.gemspec]: next time, a single case :)

File: spec/chewy/query/filters_spec.rb [hunk 2]
```diff
@@ -151,7 +151,7 @@ def query(&block)
       expect(query { (email == 'email') | (name == 'name') })
         .to be_eql Or(Equal(:email, 'email'), Equal(:name, 'name'))
     end
-    specify { expect(query { !(email == 'email') }).to be_eql Not(Equal(:email, 'email')) }
+    specify { expect(query { email != 'email' }).to be_eql Not(Equal(:email, 'email')) }
```
Comment (barthez) [inline spec/chewy/query/filters_spec.rb]: Weren't you testing how smart DSL is?

Comment (pyromaniac) [inline spec/chewy/query/filters_spec.rb]: Right, great catch

Comment (pyromaniac) [inline spec/chewy/query/filters_spec.rb]: Well, actually this one is fine

**Pull request merged**